gradle/libs.versions.toml

```diff
@@ -1,6 +1,13 @@
 [versions]
 agp = "8.6.1"
+androidxCredentials = "<latest version>"
```

Comment:
Suggestion: From what I gather online, using `<latest version>` is not necessarily a good practice, in particular for reproducibility. Shouldn't we have a hardcoded version here and update it with the other dependencies when needed?

Reply:
It should be gone now.